robocop/checkers/naming.py

```diff
@@ -662,3 +669,60 @@ def find_similar_variables(self, tokens, node):
                     col=token.col_offset,
                 )
             self.variables[normalized_token].add(token.value)
+
+
+class DeprecatedStatementChecker(VisitorChecker):
+    """Checker for finding deprecated statments."""
+
+    reports = (
+        "deprecated-statement",
+    )
+    # deprecated:alternative
+    deprecated_statements_rf3 = {}
+    deprecated_statements_rf4 = {"runkeywordunless": "IF/ELSE",
```

Comment:
IF/ELSE -> IF statement? 

So:
This keyword 'Run Keyword If' is deprecated, since version '{{ version }}', use IF statement instead

(just a thought, I'm not best on naming though :))

We can also rephrase it a bit and remove 'This keyword' (since we already point to line and col and print keyword name) and remove extra comma:
'{{ keyword_name }}' is deprecated since Robot Framework '{{ version }}', use {{ alternative }} instead

Try it out and see if it's better or not - I don't have strong opinion here